UPDATING.md

```diff
@@ -1766,6 +1767,14 @@ Now the `dag_id` will not appear repeated in the payload, and the response forma
 }
 ```
 
+## Airflow 1.10.13
+
+The HDFS hook's Kerberos support has been removed due to removed python-krbV dependency from PyPI
```

Comment:
Seems we miss a title (a short description) here.

Reply:
True. Added.